apps/meteor/server/models/raw/Users.js

```diff
@@ -1644,7 +1644,7 @@ export class UsersRaw extends BaseRaw {
 		};
 
 		if (showAgentEmail) {
-			options.fields.emails = 1;
+			options.projection.emails = 1;
```

Comment:
The if here seems a bit unnecessary here, wdyt? Perhaps we can move the logic up while the option's prop is being built... perhaps something like this:

```
const options = {
			projection: {
				name: 1,
				username: 1,
				phone: 1,
				customFields: 1,
				status: 1,
				livechat: 1,
				...(showAgentEmail && { emails: 1 }),
			},
		};
```